lib/player_module.rb

```diff
@@ -0,0 +1,14 @@
+require "human"
+require "computer"
+
+module Player
```

Comment:
I'd name the module the same as the filename (but with the camel case)

Reply:
Done. Thank you.